app.py

```diff
@@ -28,7 +28,7 @@
         'region': deployment_region,
     }
     logical_id_prefix = get_logical_id_prefix()
-    
+
```

Comment:
@bwighane why the extra space here?

Reply:
It should have been added by a file formatter and it makes sense. I can remove it as well easily.